src/main/java/com/microsoft/sqlserver/jdbc/SQLServerCallableStatement.java

```diff
@@ -257,7 +257,7 @@ boolean onDone(TDSReader tdsReader) throws SQLServerException {
         }
 
         ExecDoneHandler execDoneHandler = new ExecDoneHandler();
-        TDSParser.parse(resultsReader(), execDoneHandler);
+        TDSParser.parse(resultsReader(), execDoneHandler, false);
```

Comment:
Why explicitly add `false` parameter here? The two arg version of `TDSParser.parse(...)` has been updated to default to false and call the three arg version via `parse(..., ..., false)` so its not necessary. Ditto for the other ten places that have been updated with the new `false` parameter. If it's meant to be explicit everywhere then the old method should be removed and it should be added across the board but there are still other two-arg invocations.

Reply:
Hi @sehrope 

The existing 2 param parse call accepts String argument as second param to define `logContext` while the second method accepts `TokenHandler` instance. The new param is added only for specific case where only TDS_MSG tokens are to be read. I overloaded the original `parse` method so the driver calls will not be modified to include third parameter except where its truly desired.